types/next-auth.d.ts

```diff
@@ -0,0 +1,25 @@
+/* eslint-disable @typescript-eslint/no-unused-vars */
+import {NextAuth} from 'next-auth';
+import { JWT  } from 'next-auth/jwt';
```

Comment:
Skulle klage på linteren og spacing her, men ser at det også er ubrukt
```suggestion
```

Reply:
Faktisk ikke ubrukt, må ha det for at typingen skal fungere...aner ikke hvorfor det er slik